src/core/a-animation.js

```diff
@@ -356,13 +357,14 @@ function getAnimationValues (el, attribute, dataFrom, dataTo, currentValue) {
   var from = {};
   var partialSetAttribute;
   var to = {};
-
-  if (attributeSplit.length === 2) {
+  if (attributeSplit.length === 2 && attributeSplit.indexOf('color') === -1) {
```

Comment:
Ok so exact matching the scheme type should be safe here and then the rest should still work as far as the tweets are concerned.  I'm using the passed attributed to set when there are more than one in attributeSplit